### PR TITLE
Morse bonded interaction

### PR DESCRIPTION
**Pull request by chemicalfiend**
Added specific interaction type morse with force and potential energy functions. Parameterised with depth D and parameters alpha and r0.

### CODE REVIEW

File: src/interactions/morse.jl [hunk 1]
```diff
@@ -0,0 +1,32 @@
+export Morse
+
+"""
+    Morse
+
+Morse potential for bonds between two atoms
+
+"""
+
```
Comment (jgreener64) [inline src/interactions/morse.jl:9]: Remove this line.

File: src/interactions/morse.jl [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+struct Morse{T, A, R} <: SpecificInteraction
```
Comment (jgreener64) [inline src/interactions/morse.jl:10]: Maybe call it `MorseBond` to match `HarmonicBond`?

File: src/interactions/morse.jl [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    D::T
+    α::A
+    r0::R
+end
+
```
Comment (jgreener64) [inline src/interactions/morse.jl:15]: Add a constructor using keyword arguments here like for `HarmonicBond`.

**Review by jgreener64 — COMMENTED** [review]
Looks good - left some minor comments.

File: src/interactions/morse.jl (hunk 2 above)
Comment (chemicalfiend) [reply src/interactions/morse.jl]: Done.

File: src/interactions/morse.jl (hunk 3 above)
Comment (chemicalfiend) [reply src/interactions/morse.jl:14]: Forgot that initially. Took care of it, it's done.

**Comment by jgreener64** [comment]
:+1:

**Pull request merged**